jujugui/static/gui/src/test/test_browser_charm_details.js

```diff
@@ -330,9 +330,11 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
     it('_addCharmEnvironment displays the config panel', function(done) {
-      var fakeStore = new Y.juju.charmstore.APIv4({
-        charmstoreURL: 'localhost/'
-      });
+      window.juju_config = {
```

Comment:
Is this sufficient? https://github.com/kadams54/juju-gui/blob/charm-details/jujugui/static/gui/src/test/test_browser_charm_details.js#L77